auth0/v3/test/management/test_rest.py

```diff
@@ -198,6 +198,56 @@ def test_post_error_with_no_response_text(self, mock_post):
             self.assertEqual(context.exception.error_code, 'a0.sdk.internal.unknown')
             self.assertEqual(context.exception.message, '')
 
+    @mock.patch('requests.post')
+    def test_post_error_with_error_but_no_error_code(self, mock_post):
+        rc = RestClient(jwt='a-token', telemetry=False)
+
+        for error_status in [400, 500, None]:
+            mock_post.return_value.status_code = error_status
+            mock_post.return_value.text = '{"error": "SomeError"}'
```

Comment:
Our errors typically come back with both an `error` property and an `error_description` one. I don't know OTOH but I'm guessing the `exception` that's returned will include the description as an error message. Might be good to check for that as well.